docs/requirements.txt

```diff
@@ -2,4 +2,5 @@ ipython
 scipy
 numpy > 1.9.0
 scikit-learn >= 0.18
+pandas
```

Comment:
Fixed in e5f4fba
> it seems ipython is in requirements.txt because for IPython mock didn't work for some reason. But I'm not sure.

Yes, just noticed, likely you already tried to mock it :)